azurerm/internal/services/mixedreality/spatial_anchors_account_data_source_test.go

```diff
@@ -0,0 +1,51 @@
+package mixedreality_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+)
+
+type SpatialAnchorsAccountDataSource struct {
+}
+
+func TestAccSpatialAnchorsAccountDataSource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_spatial_anchors_account", "test")
+	r := SpatialAnchorsAccountDataSource{}
+
+	data.DataSourceTest(t, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("name").Exists(),
```

Comment:
Can we further check the existance of the `account_id` and the `account_domain`? Also applies to the test cases for the managed resource.